chain_info/Moonbase.yaml

```diff
@@ -0,0 +1,54 @@
+name: Moonbase Alpha
```

Comment:
This must match the file name. So you need to edit the filename or this value.

Reply:
file renamed